src/main/java/com/erstudio/ERSStudioReleaseHelperUI.java

```diff
@@ -0,0 +1,239 @@
+package main.java.com.erstudio;
+
+/**
+ * @author akshit.arora
+ * for the UI Skeleton
+ */
+import main.java.com.erstudio.constants.Constants;
+import javax.swing.*;
+import javax.swing.filechooser.FileSystemView;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class ERSStudioReleaseHelperUI {
+
+    JLabel labelTSString, labelVersionString, labelNewVersionString, labelGUID;
+    JTextField textFieldTs, textVersionString, textNewVersionString, textGUID;
+    JButton buttonBrowse, buttonGenerate, buttonChangeVersion, buttonReset;
+    ButtonGroup buttonGroupUpgradeType;
+    JRadioButton jRadioButtonFullUpgrade, jRadioButtonPatchUpgrade;
```

Comment:
use radioButtonFullUpgrade. like you are using buttonBrowse, textFieldTs.

Reply:
done